CMakeLists.txt

```diff
@@ -31,11 +34,21 @@ message(STATUS "Building xeus-r v${${PROJECT_NAME}_VERSION}")
 # =============
 
 option(XEUS_R_BUILD_STATIC "Build xeus-r static library" ON)
-OPTION(XEUS_R_BUILD_SHARED "Split xr build into executable and library" ON)
-OPTION(XEUS_R_BUILD_EXECUTABLE "Build the xr executable" ON)
-
-OPTION(XEUS_R_USE_SHARED_XEUS "Link xr  with the xeus shared library (instead of the static library)" ON)
-OPTION(XEUS_R_USE_SHARED_XEUS_R "Link xr  with the xeus shared library (instead of the static library)" ON)
+option(XEUS_R_BUILD_SHARED "Split xr build into executable and library" ON)
+option(XEUS_R_BUILD_EXECUTABLE "Build the xr executable" ON)
+
+option(XEUS_R_USE_SHARED_XEUS "Link xr  with the xeus shared library (instead of the static library)" ON)
+option(XEUS_R_USE_SHARED_XEUS_R "Link xr  with the xeus shared library (instead of the static library)" ON)
```

Comment:
The documentation is the same for both options, which I guess is not what we expect. `XEUS_R_USE_SHARED_XEUS_R` seems to mean the xeus-r executable should link with the shared xeus-r library.